html/about.html

```diff
@@ -0,0 +1,143 @@
+    <link rel="stylesheet" href="../css/index.css">
+    <link rel="stylesheet" href="../css/about.css">
+    <script src="../js/about.js" type="module" async></script>
+    <title>About - Creative Commons Global Summit 2015</title>
+  </head>
+  <body>
+    <main id="main" class="container-fullwidth">
+      <header>
+        <nav id="nav" class="container-fullwidth navbar">
+          <div class="desktop-visible" id="top-nav" >
+            <a href="https://facebook.com"><img src="../assets/icons/facebook.png" alt="Facebook logo"></a>
+            <a href="https://twitter.com"><img src="../assets/icons/twitter.png" alt="Twitter logo"></a>
+            <a href="#">Logout</a>
+            <a href="#">Mypage</a>
+            <a href="#">English</a>
+          </div>
+          <img src="../assets/icons/mobile-menu.png" alt="Menu hamburger button" id="hamburger" class="nav-item mobile-visible">
+          <div class="nav-options row-fullwidth mobile-visible" id="bottom-nav">
+            <a class="desktop-visible" id="logo" href="../index.html"><img src="../assets/images/logo-clickable.png" alt="Clickable logo"></a>
+            <img class="mobile-visible close-menu" src="../assets/icons/close-icon.png" alt="Close menu button.">
+            <div class="mobile-menu">
+              <a class="nav-option current-page" href="../index.html">Home</a>
+              <a class="nav-option" href="../index.html#programs">Program</a>
+              <a class="nav-option mobile-visible" href="../index.html#join-summit">Join</a>
+              <a class="nav-option desktop-visible" href="../index.html#seewhole">Join</a>
+              <a class="nav-option mobile-visible" href="#sponsors">Sponsor</a>
+              <a class="nav-option desktop-visible" href="../index.html#sponsors">Sponsor</a>
+              <a class="nav-option" href="https://creativecommons.org/tag/news/">News</a>
+              <a class="nav-option" href="https://creativecommons.org/campaign/">CC Campaign</a>
+            </div>
+          </div>
+        </nav>
+        <div id="path-icon" class="desktop-visible">
+          <a href="../index.html">
+            <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="black" class="bi bi-house" viewBox="0 0 16 16">
+              <path fill-rule="evenodd" d="M2 13.5V7h1v6.5a.5.5 0 0 0 .5.5h9a.5.5 0 0 0 .5-.5V7h1v6.5a1.5 1.5 0 0 1-1.5 1.5h-9A1.5 1.5 0 0 1 2 13.5zm11-11V6l-2-2V2.5a.5.5 0 0 1 .5-.5h1a.5.5 0 0 1 .5.5z"/>
+              <path fill-rule="evenodd" d="M7.293 1.5a1 1 0 0 1 1.414 0l6.647 6.646a.5.5 0 0 1-.708.708L8 2.207 1.354 8.854a.5.5 0 1 1-.708-.708L7.293 1.5z"/>
+            </svg>
+          </a>
+           > <a href="#">About</a>
+        </div>
+      </header>
+      <section id="about-event" class="container-fullwidth">
+        <span class="pre-title mobile-visible">"Hello! Sharing World"</span>
+        <h1 class="title">Creative Commons Global Summit 2015</h1>
```

Comment:
- [ ] Please increase the size of this heading and make sure it matches the design on the template.

### Your design

![image](https://user-images.githubusercontent.com/83803414/163253958-d309d186-7d33-487c-b37e-d179de5e10ea.png)

### Behance design

![image](https://user-images.githubusercontent.com/83803414/163253995-d482eabe-ca03-4da8-9cd5-4f44a8c2993a.png)